modules/dnn/src/layers/slice_layer.cpp

```diff
@@ -188,6 +205,8 @@ class SliceLayerImpl : public SliceLayer
         const MatSize& inpShape = inputs[0].size;
 
         finalSliceRanges = sliceRanges;
+        finalSliceSteps = sliceSteps;
```

Comment:
Do we really need `finalSliceSteps`? We use `finalSliceRanges` to replace negative ranges to absolute values depends on input shape. Do we have undetermined steps?